CMakeLists.txt

```diff
@@ -0,0 +1,130 @@
+# CMake version configuration
+# (Note: The minimal version 3.5 is shipped with Ubuntu 16.04)
+cmake_minimum_required(VERSION 3.5...3.19)
+if(${CMAKE_VERSION} VERSION_LESS 3.12)
+    cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
+endif()
+
+# Define project
+project(librepcb
+    VERSION 0.1.5
```

Comment:
Yes, I think we can inject the CMake version into the cpp file!

Reply:
Actually it's a bit complicated, because the `VERSION` must be in the format `VERSION <major>[.<minor>[.<patch>[.<tweak>]]]`, so the `-unstable` version specifier is not valid. However, the version is optional, so we can remove it from CMake, and use our own variable instead, when accessing the version.

Reply:
It's now moved into the top-level `CMakeLists.txt`.

Reply:
That's nice, but now this comment: 

```cpp
// Read the release workflow documentation (at https://developers.librepcb.org)
// before making changes here!!!
```

is still located in `application.cpp` although this is no longer the place where the version number needs to be adjusted :wink: So this comment should be moved into `CMakeLists.txt` as well.

In addition, the whole "Version Information" block containing the constants `APP_VERSION`, `FILE_FORMAT_VERSION` and `FILE_FORMAT_STABLE` (in `application.cpp`) is no longer needed and could/should be removed. Just use the new defines from `build_env.h` directly where the old ones were used previously.

Reply:
Good point, fixed.